nion/swift/model/Utility.py

```diff
@@ -432,6 +433,50 @@ def do_sample() -> None:
     threading.Thread(target=do_sample).start()
 
 
+def simplify_filename(filename: str, replacement_char: str = '_', maximum_length: int = 64) -> str:
```

Comment:
I'm questioning the limit of 64 - this seems too short and maybe unnecessary. Is there a reason for this? macOS limit is 255 utf-8 characters. Windows has different restrictions, seemingly 32K when using Unicode calls (see https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry). Linux is more flexible.

Reply:
That would be the third task in #591

Reply:
Also this is only the file name - it does not include the full path so we need to consider what a sensible maximum might be. That said, it is easy to change the default value once we make a decision.